src/main/java/bio/terra/pdao/gcs/GcsProjectFactory.java

```diff
@@ -17,13 +17,10 @@ public GcsProjectFactory(GcsConfiguration gcsConfiguration) {
     }
 
     public GcsProject get(String projectId) {
-        if (!gcsProjectCache.containsKey(projectId)) {
-            GcsProject gcsProject = new GcsProject(
-                projectId,
+        gcsProjectCache.computeIfAbsent(projectId, (p) -> new GcsProject(
```

Comment:
For single-arg lambdas you don't need `()`s
```suggestion
        gcsProjectCache.computeIfAbsent(projectId, p -> new GcsProject(
```